plugins/module_utils/foreman_helper.py

```diff
@@ -325,9 +325,10 @@ def run(self, **kwargs):
                 ak_param = {'name': 'kt_activation_keys', 'parameter_type': 'string', 'value': self.foreman_params.pop('activation_keys')}
                 self.foreman_params['parameters'] = parameters + [ak_param]
```

Comment:
I think, it can still happen here, that you end up with doubled "kt_activation_keys" if the user specified it in both ways.
Arguably, this is not a proper way to call the module.

Reply:
If the user specified it in both places, they have earned the error.

Reply:
:trumpet: